MapboxDirections/MBRouteLeg.swift

```diff
@@ -98,6 +130,9 @@ open class RouteLeg: NSObject, NSSecureCoding {
      */
     open let steps: [RouteStep]
     
+    
+    open let annotation: [AnnotationType:[Any]]?
```

Comment:
A little trivia: "[segment](https://wiki.openstreetmap.org/wiki/Segment)" used to be a primitive data type in OpenStreetMap back in the day.